docs/guides/operator/advanced-configuration.adoc

```diff
@@ -349,7 +349,7 @@ NOTE: The `tracing-jdbc-enabled` is not promoted as a first-class citizen as it
 
 For more details about tracing, see <@links.observability id="tracing" />.
 
-=== Network Policies (Experimental)
+=== Network Policies (Preview)
```

Comment:
Thank you for the review. Let's go ahead and merge it then, but continue the preview/experimental part of it.

I don't think we have the feature of supported/unsupported, as we don't have feature flags in the Operator. At some point we were discussing that a feature flag could steer functionality in Keycloak and the KC Operator, but never only in the Operator. 

We could move it in the CR below an explicit "unsupported" like there is for the pod template, but then people would move it again once it is supported.

BTW, I think this is quite straightforward and feature complete, I don't expect any changes, and it will be supported in KC26.2 AFAIK (although not enabled by default due to not having any breaking changes). If someone could find a way to enabled and declare it as a non-breaking change, I'm all ears!

The issue to make it supported: #36036

Reply:
To add my two cents, preview allows us to change the CRD syntax if we get a suggestion or feedback from users using this feature in KC 26.1.
I'm confident the feature is completed and no major change is required for KC 26.2 (except removing the "preview" from the docs and making it default).